fe/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -236,11 +236,11 @@ public void analyze(Analyzer analyzer) throws UserException {
             // if all previous job finished
             UUID uuid = UUID.randomUUID();
             String jobLabel = "insert_" + uuid;
-            LoadJobSourceType sourceType = isStreaming ? LoadJobSourceType.INSERT_STREAMING 
+            LoadJobSourceType sourceType = isStreaming ? LoadJobSourceType.INSERT_STREAMING
                     : LoadJobSourceType.FRONTEND;
             transactionId = Catalog.getCurrentGlobalTransactionMgr().beginTransaction(db.getId(),
                     jobLabel,
-                    "fe", sourceType);
+                    "fe", sourceType, null);
```

Comment:
Like this, you can add a `beginTransaction` with four parameters, and old version function call new version add a `null` argument.